ALUM/ALUM/Components/FilledButton.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  FilledButton.swift
+//  ALUM
+//
+//  Created by Yash Ravipati on 1/18/23.
+//
+
+import SwiftUI
+
+struct FilledInButton : ButtonStyle{
+    
+    @State var isLarge: Bool = false
+    @State var disabled: Bool = false
+    
+    var largeWidth = 358.0
+    var smallWidth = 104.0
+    
+    func makeBody(configuration: Configuration) -> some View {
+            if(!disabled){
+                configuration.label
+                    .frame(width: isLarge ? largeWidth : smallWidth)
+                    .frame(height: 48)
+                    .foregroundColor(.white)
+                    .font(.system(size: 17, weight: .medium))
```

Comment:
Instead of .system(size: 17, weight: .medium) can we use .body to support Apple's dynamic type. 

Apple has this feature called dynamic type which lets users set their preferred font sizes (out of few categories) which are then used across most apps. 

Check this out for understanding dynamic types (especially the table) - 
https://developer.apple.com/design/human-interface-guidelines/foundations/typography/#:~:text=xxxLarge-,Large%20(Default),13,-Point%20size%20based